lib/emails/conversationUpdate.tsx

```
@@ -0,0 +1,101 @@
+import { Body, Head, Hr, Html, Img, Link, Preview, Text } from "@react-email/components";
```

ESLint

[error] 'Img' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)